pkgs/by-name/ju/jump/package.nix

```
@@ -27,6 +30,11 @@ buildGoModule rec {
 
   postInstall = ''
     installManPage man/j.1 man/jump.1
+
+    installShellCompletion --cmd jump \
+       --bash <($out/bin/jump shell bash) \
+       --fish <($out/bin/jump shell fish) \
+       --zsh <($out/bin/jump shell zsh) \
```

```suggestion
  '' + lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
    installShellCompletion --cmd jump \
       --bash <($out/bin/jump shell bash) \
       --fish <($out/bin/jump shell fish) \
       --zsh <($out/bin/jump shell zsh)
```
https://nixos.org/manual/nixpkgs/unstable/#installshellfiles-installshellcompletion-exampleusage-guarded